prime-router/metadata/schemas/STRAC/strac-covid-19.schema

```diff
@@ -112,14 +112,52 @@ elements:
     csvFields: [{name: patient_zip, format: $zipFive}]
 
   - name: test_result
```

Comment:
Bug fix:  The fake data generator was creating code values that the minimalist alt valueset didn't have.

As long as strac continues to only send the first 3 values, the newly added values should be harmless.